client/my-sites/site-settings/section-writing.jsx

```diff
@@ -20,9 +20,9 @@ module.exports = React.createClass( {
 	render: function() {
 
 		return (
-			<Card className="writing-settings">
-				<WritingForm site={ this.props.site } />
-			</Card>
+
+			<WritingForm site={ this.props.site } />
```

Comment:
`Card` is no longer necessary after these edits, so it should be removed in [Line 10](https://github.com/Automattic/wp-calypso/pull/1898/files#diff-101df1adbb8b8cdb6f43fd26a61e8a64L10)